### PR TITLE
Make blank issue a more prominent option

### PR DESCRIPTION
**Pull request by GMNGeoffrey**
Collaborators keep creating issues with the templates when they really
should be doing triage and stuff themselves and we end up with a bunch
of boilerplate, incorrect labels, etc.

### CODE REVIEW

**Comment by GMNGeoffrey** [comment]
This demonstrates two different options for making the blank issue more
prominent:

1. An "external link" that just points to the blank issue URL
2. A template that barely modifies the blank issue URL

External link:
  - Pros:
    - less configuration
    - blank issue is still sorted at the bottom (apparently there's no
      way to sort issue templates lower, though I could probably make 
      blank the *last* issue template by adding a "z" prefix)
  - Cons:
    - it's not really external, so having it as an external link looks
      a bit weird.
    - It's hard-coded to this specific repository URL
    - ? Opens issue template in a new tab
 
Issue template:
  - Pros:
    - We can add reminders for people to triage their issues
  - Cons:
    - Has to be sorted above external links
    - Sorting below other templates requires naming shenanigans
    - More configuration

Thoughts?

Demo of both on my fork: https://github.com/GMNGeoffrey/iree/issues/new/choose

Issue chooser:
issue_chooser

Blank issue custom template:
blank_issue_template

A real blank issue:
regular_blank_issue

File: .github/ISSUE_TEMPLATE/blank_issue.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: 📄 Blank Issue
+description: If you know what you're doing (especially collaborators)
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please remember to add appropriate labels, assignees, projects, etc.
```
Comment (ScottTodd) [inline .github/ISSUE_TEMPLATE/blank_issue.yml:7]: I think I prefer this custom template, even if the sorting is not quite what we want.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -9,3 +9,6 @@ contact_links:
   - name: 💬 Join us on Discord
     url: https://discord.gg/26P4xW4
     about: For realtime communication with the team and collaborators
+  - name: 📄 Blank Issue
+    url: https://github.com/iree-org/iree/issues/new
+    about: If you know what you're doing (especially collaborators)
```
Comment (ScottTodd) [inline .github/ISSUE_TEMPLATE/config.yml:14]: (reminder to remove one or the other before merge)

**Review by ScottTodd — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/blank_issue.yml (hunk 1 above)
Comment (GMNGeoffrey) [reply .github/ISSUE_TEMPLATE/blank_issue.yml:7]: :+1: Should I call it "z_blank_issue.yml" for sorting? A bit gross, but I think the user experience is more important here

Comment (ScottTodd) [reply .github/ISSUE_TEMPLATE/blank_issue.yml:7]: Oh... hmm.. if the name of the _file_ controls sorting, but we can still set the `name` in yml to whatever we want independent of that, then sure... I guess :P

Comment (GMNGeoffrey) [reply .github/ISSUE_TEMPLATE/blank_issue.yml:7]: That's my guess. Not sure though

Comment (GMNGeoffrey) [reply .github/ISSUE_TEMPLATE/blank_issue.yml:7]: Indeed it does:

image

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (GMNGeoffrey) [reply .github/ISSUE_TEMPLATE/config.yml]: Done (I appreciate these reminders, btw :slightly_smiling_face:)

**Pull request merged**